Makefile

```diff
@@ -59,7 +59,7 @@ endif
 
 
 ifeq ($(BUILD_OMS),BUILD_OMS)
-dsckit098: nx nxOMSPerfCounter nxOMSSyslog nxOMSKeyMgmt nxOMSPlugin nxOMSCustomLog nxFileInventory nxOMSGenerateInventoryMof nxOMSAgentNPMConfig nxOMSAutomationWorker nxOMSAuditdPlugin
+dsckit098: nx nxOMSPerfCounter nxOMSSyslog nxOMSKeyMgmt nxOMSPlugin nxOMSCustomLog nxOMSCustomLogNew nxFileInventory nxOMSGenerateInventoryMof nxOMSAgentNPMConfig nxOMSAutomationWorker nxOMSAuditdPlugin
```

Comment:
How much time would it take to change "nxOMSCustomLogNew" to "nxOMSSudoCustomLog"?
If it would take less than two hours to make the change and test once, then you may want to consider it.
Else, it's not too important. Maybe add a comment at the top of the files as to what the meaning of "new" is.